segmentation_models_pytorch/encoders/timm_sknet.py

```diff
@@ -74,7 +74,7 @@ def load_state_dict(self, state_dict, **kwargs):
             'block': SelectiveKernelBasic,
             'layers': [2, 2, 2, 2],
             'zero_init_last_bn': False,
-            'block_args': {'sk_kwargs': {'min_attn_channels': 16, 'attn_reduction': 8, 'split_input': True}}
+            'block_args': {'sk_kwargs': {'rd_ratio': 1/8, 'split_input': True}}
```

Comment:
1. `attn_reduction` was changed to `rd_ratio` and the value is inversed. So old `attn_reduction` equals to new `1 / rd_ratio`

See https://github.com/rwightman/pytorch-image-models/commit/307a935b790b5af8d551ebecda053cb1a9b16fcb#diff-69a503e8326443f3698de117aa9ac39e8fbb9ce52dfa7783c6708fe6a38f6e87R52

2. There is no more `min_attn_channels`. I did not find an easy way to keep it here and I think it is fine to just delete it.  

See https://github.com/rwightman/pytorch-image-models/commit/bda8ab015ac5ee0ec75b8c59d5c0c3b399abda94#diff-69a503e8326443f3698de117aa9ac39e8fbb9ce52dfa7783c6708fe6a38f6e87R52